### PR TITLE
feat: update robots.txt

### LINKED ISSUE #6283: Disallow indexing few pages for crawling robots
Currently, we are fighting to get good content on Google SEO and be in bit control; we got an advisory to drop a lot of "bad" pages which are not interesting for bots and raise our quality for them and get speedy-turn-around-reindex times.

image

Pages we should disallow for robots
---
- items - like URLs with "/details/" & "/gallery/"
- profiles - "/u/"
- all prefixes with "beta.", "canary", "nft.", "form.", "*.w." (like polysearch, image, image-beta, )  
- transfer - "/transfer/"

beta is currently 60% URLs
nft. has fair portion

we will follow up with pre-render integration as I sense it's culprit CF doesn't do it.

Ref
---
https://support.google.com/webmasters/answer/7440203#crawled

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #6283
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=12zQP2JviSyZzAX2j31F2TR5uSvDNerruSJBWxEZXjW3KMaN)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at e0dff1c</samp>

Updated `robots.txt` to block unwanted crawlers and optimize site visibility.

### <samp>🤖 Generated by Copilot at e0dff1c</samp>

> _`robots.txt` blocks_
> _paths and subdomains to hide_
> _autumn leaves falling_

### CODE REVIEW

File: static/robots.txt [hunk 1]
```diff
@@ -1,2 +1,10 @@
 User-agent: *
-Disallow:
+Disallow: */details/
+Disallow: */gallery/
+Disallow: */u/
+Disallow: */transfer/
```
Comment (Jarsen136) [inline static/robots.txt:5]: The url path of the transfer page is `/{prefix}/transfer` without the ending `/`

image

**Review by Jarsen136 — COMMENTED** [review]
robots.txt guide
ref: https://developers.google.com/search/docs/crawling-indexing/robots/robots_txt#syntax

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
what happened with tests 

image

**Comment by floyd-li** [comment]
> what happened with tests
> 
> image

hmm not sure what's happened. it should not be related with this pr 🤔️

**Comment by Jarsen136** [comment]
image
I guess it should be related to this change
https://github.com/kodadot/nft-gallery/pull/6265/files

Could u pls take a look? @preschian

**Review by vikiival — APPROVED** [review]

**Review by roiLeo — APPROVED** [review]
Missing:
-`*/collection/`
- sitemap

otherwise lgtm

**Comment by preschian** [comment]
> I guess it should be related to this change [#6265 (files)](https://github.com/kodadot/nft-gallery/pull/6265/files)
> 
> Could u pls take a look? @preschian

if still error I think we need to downgrade the version again in the meantime
https://github.com/Wandalen/wretry.action/issues/93

**Comment by floyd-li** [comment]
> Missing:
> 
>     * `*/collection/`
> 
>     * sitemap
> 
> 
> otherwise lgtm

updated

**Comment by Jarsen136** [comment]
> Missing:
> 
> * `*/collection/`

I'm not sure if we are going to disallow the collection details page. @yangwao

**Comment by yangwao** [comment]
> I'm not sure if we are going to disallow the collection details page. @yangwao

for now yes probably, later we will use `allow` option to list top10 which has liquidity tbh, time to play gods game as still learning what is best tactic as "e-shop" what to index and what to not. 

The current goal is to have top10 great pages to show, rather 1000 sub-optimal ones. 

We will be selecting pages to put online up 
- https://github.com/kodadot/ops-internal/issues/587

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.66 USD/DOT ~ 6.438 $DOT
🧗 [12zQP2JviSyZzAX2j31F2TR5uSvDNerruSJBWxEZXjW3KMaN](https://polkadot.subscan.io/account/12zQP2JviSyZzAX2j31F2TR5uSvDNerruSJBWxEZXjW3KMaN)
🔗 [0x75ab994221c585009cbc1cff103beedde4ae02f49197259cb36362868b7582f1](https://polkadot.subscan.io/extrinsic/0x75ab994221c585009cbc1cff103beedde4ae02f49197259cb36362868b7582f1)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**